docs/configuration.md

```diff
@@ -934,7 +934,16 @@ The `silly` auth is only for development purposes. It simply checks for the
 existence of the `Authorization` header in the HTTP request. It has no regard for
 the header's value. If the header does not exist, the `silly` auth responds with a
 challenge response, echoing back the realm, service, and scope that access was
-denied for.
+denied for.  The silly auth endpoint should return a jwt per https://github.com/docker/distribution/blob/master/docs/spec/auth/token.md in order
```

Comment:
Silly auth does none of this. Silly auth is a scheme to be used for development and unit testing. If you need an example to configure it, you probably shouldn't be using it.